### PR TITLE
[SUCE-480] Fix carousel on firefox

### PR DESCRIPTION
**Pull request by TrebuhD**

### CODE REVIEW

File: client/elements/addons/text-carousel.html [hunk 1]
```diff
@@ -89,14 +90,26 @@
 
     </style>
 
+    <iron-ajax
+        auto
```
Comment (SamaneriVimala) [inline client/elements/addons/text-carousel.html]: is `auto` the best here?

File: client/localization/extract_static_strings.py [hunk 2]
```diff
@@ -32,6 +32,8 @@ def replace_attrs(element, data):
 def hash_element_content(element, data):
     string = ''.join((str(child) for child in element.children))
     string = ' '.join(string.split())
+    if string == '':
+        return
```
Comment (SamaneriVimala) [inline client/localization/extract_static_strings.py]: semicolon

File: client/elements/addons/text-carousel.html (hunk 1 above)
Comment (TrebuhD) [reply client/elements/addons/text-carousel.html]: Good point. I'll check tomorrow if it hinders performance.

File: client/localization/extract_static_strings.py (hunk 2 above)
Comment (TrebuhD) [reply client/localization/extract_static_strings.py]: Python file! Fooled you ;)

**Pull request merged**